rules-tests/Php74/Rector/Property/TypedPropertyRector/FixtureUnionIntersectionTypes/mixed_type.php.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Rector\Tests\Php74\Rector\Property\TypedPropertyRector\FixtureUnionIntersectionTypes;
+
+class SomeMixedType
+{
+  /** @var mixed */
+  private $x;
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php74\Rector\Property\TypedPropertyRector\FixtureUnionIntersectionTypes;
+
+class SomeMixedType
+{
+  private mixed $x;
```

Comment:
This file is in the Rector\Tests\Php74 namespace.

But, as discussed in the other PR, mixed doesn't exist in PHP 7.4. Results in an error.

Shouldn't there be two tests? 

- One in: Rector\Tests\Php74 - no change
- Another in: Rector\Tests\Php80 - this test